Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Enable RPATH also on Hurd
 Enable RPATH also on the Hurd, so the spectrum example can locate the fftreal
 library in its own directory.
 .
 The "hurd-*" addition can be upstreamed.
Last-Update: 2023-07-14
Forwarded: no

--- a/examples/multimedia/spectrum/app/app.pro
+++ b/examples/multimedia/spectrum/app/app.pro
@@ -78,7 +78,7 @@
                                 ../spectrum.app/Contents/MacOS/spectrum
     }
 } else {
-    linux-g++*: {
+    linux-g++*|hurd-*: {
         # Provide relative path from application to fftreal library
         QMAKE_LFLAGS += -Wl,--rpath=\\\$\$ORIGIN
     }
